tests/phpunit/CRM/Contribute/Form/Search.php

```diff
@@ -167,4 +167,109 @@ public function testBatchFilter() {
     }
   }
 
+  /**
+   *  CRM-20286: Test CRM_Contribute_Form_Search Card type filters
+   */
+  public function testCardTypeFilter() {
+    $this->quickCleanup($this->_tablesToTruncate);
+    $contactID1 = $this->individualCreate(array(), 1);
+    $contactID2 = $this->individualCreate(array(), 2);
+    $Contribution1 = civicrm_api3('Contribution', 'create', array(
```

Comment:
minor point, in the tests it's better to use $this->callAPISuccess - it has better handling when there is a bug